File: app/src/main/java/com/example/q/myapplication/MainActivity.java
package com.example.q.myapplication;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }
    public  void on_admin(View v){
        Intent admin =new Intent(MainActivity.this,admin.class);
        startActivity(admin);
    }
    public  void on_new(View v){
        Intent new_1 =new Intent(MainActivity.this,newT.class);
        startActivity(new_1);
    }
    public  void on_curr(View v){
        Intent curr =new Intent(MainActivity.this,current.class);
        startActivity(curr);
    }
}

File: app/src/main/java/com/example/q/myapplication/MainActivity.kt
package com.example.q.myapplication

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    fun on_admin(v: View) {
        
        var admin= Intent(this,MainActivity::class.java)
        startActivity(admin)
        
    }

    fun on_new(v: View) {
        var admin= Intent(this,MainActivity::class.java)
        startActivity(admin)
        val new_1=Intent(this@MainActivity, newT::class.java)
        startActivity(new_1)
       
    }

    fun on_curr(v: View) {
        val curr=Intent(this@MainActivity, current::class.java)
        startActivity(curr)
    }
}
